cli.go

```diff
@@ -3,7 +3,6 @@ package cli
 import (
 	"encoding/json"
 	"fmt"
-	"log"
```

Comment:
This wasn't correct, lines like 55 and 58 were indirectly using another log package.